### ISSUE TITLE
Release 19.04

### ISSUE THREAD

**Issue by skmp** [reicast/reicast-emulator#1522]
# Pending tasks
(for skmp)
- [ ] Fix android tv certification
- [ ] Coordinate w/ fh for final touches
- [ ] Merge `fh/mymaster`
- [ ] Test the release

# Testing
Testing ticket: #1518 

# Proposed timeline
- Test things, finalize commits & merge before the weekend
- Have final RC on 31st of march
- Push to early adopters & test manually
- Tag r19.04 on April 1st, nullDC's 12th birthday.

# Notes
- Log script : `git log --merges --pretty=format:"- %b [%h, %an]" --abbrev-commit r7..r8`. Add to some release script
- Remember to make an annotated tag, `git tag -a r19.04`
- Release channels: Github releases, Android Store, snapcraft (?), others (?)
- Targeting platforms: armv7, arm64, x86, x64

**Comment by spielvan** [comment]
skmp is very good builds, I tested cb278e3 and the result was very satisfactory, but I continue with the bug in the Mortal Kombat Gold game, in the scenario of Goro, which cuts half the screen, this same bug does not occur in the fh / mymaster build and fh / content-browser, a request that I have, would be able to install several builds on android? currently to install a different build need to uninstall the previous one, and if it could have several installed could compare each one of them. A hug.

**Comment by skmp** [comment]
Merged #1523 (past year’s WIP from FH)

**Comment by skmp** [comment]
@flyinghead looks like the merge also broke snapcraft, `/usr/bin/ld: cannot find -ludev` https://build.snapcraft.io/user/reicast/reicast-emulator/506975.

**Comment by skmp** [comment]
Updated the beta support link to chat.reicast.com from #1208, as that seems like a much more appropriate point. Also made a post on FB to test the post-merge version :)

**Comment by mar753** [comment]
My changes like odd screen resolution handling, scaling, lowering rendering resolution, fast RTT are gone after fh/master->master merge

**Comment by skmp** [comment]
Figured the merge might have disrupted things, that's why I poked on the other issue. Which MRs were they? #1481 #1480 #1479 and #1460 ? Are you on discord?

**Comment by mar753** [comment]
Yep. Gles files were overwritten so there's no odd screen resolution handling, scaling and fast RTT.

**Comment by spielvan** [comment]
@skmp This new menu of choice of games has a bug that I had commented with @flyinghead, if I am using an xbox360 gamepad by usb x-input in my box android tv, it detects the game pad as control 2 and control one is for the touchscreem, so every time you open reicast, you need a mouse to enable the gamepad, it would be interesting not to need the mouse and the gamepad will load when you open reicast.

**Comment by Emulatorer** [comment]
@skmp tommorow is the next official update in google play store?
